docs/general/builtins/scalar-functions.rst

```diff
@@ -2614,6 +2617,18 @@ Returns: ``integer``
     +------+
     SELECT 1 row in set (... sec)
 
+If there is at least one empty array or ``NULL`` on the requested dimension
```

Comment:
Not sure - with upper it was clear that we go for max but how we compare NULL and 1? Cannot also align with PG as they don't allow mixing sizes so all sibling arrays on some dimension are same, in our case we have to choose.

Reply:
sounds okay to me to do it like this.